konsave/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Top-level Konsave package."""
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    # Package is not installed
+    pass
```

Comment:
I refactored the logic for getting the package's version. Now the package (the package's `__init__.py`) provides a `__version__` attribute, which is a common and expected way of providing info on the package's version.  This is also done dynamically, so there won't be a need to manually update a `VERSION` constant :wink:.

Example (from an ipython session):
```python
In [1]: import konsave

In [2]: konsave.__version__
Out[2]: '1.1.10.dev1+ga506ace'
```

@Prayag2 please resolve this comment if you agree on this approach.

Reply:
Thank you so much for this. I actually wanted to do this but didn't know how.